### DISCUSSION TITLE
[freshly installed lazyvim] "K", or hover, shows doc strings in a new tab, rather than in a hover window

### DISCUSSION THREAD

**Discussion by hopezh** [LazyVim/LazyVim#4420] [Q&A]
As shown in the video below, using "K", or hover, opens a new tab to show the doc strings of `numpy`, when I'expecting a hover window just besides the word numpy, and this happens in a freshely installed lazyvim. 

Also, there's no K for hover by searching keymappings, as shown in second video. 

May I ask if anyone has encounter such issue, and how to solve it? Thank. 

https://github.com/user-attachments/assets/a7e4a26c-fa8d-4373-a958-100c32b2306c

https://github.com/user-attachments/assets/265eb834-5688-448e-84be-90a623728b9c

**Comment by dpetka2001** [comment]
This is not default LazyVim behavior. Following a screencast installing LazyVim from scratch and how it is with only Python Extra enabled.

[Screencast_2024-09-09-19-40-57.webm](https://github.com/user-attachments/assets/f245fd28-5e04-48bf-a9a9-4f92812d615c)

**Comment by VicPetrov** [comment]
Have the same issue, never explicitly tried to override this behaviour.

**Comment by VicPetrov** [comment]
It worked fine before the recent updates with the fzf-lua etc.

**Reply by dpetka2001** [reply]
If there's no LSP client attached to the buffer, then `K` defaults to `:help`. If there's a LSP client attached to the buffer, then `K` is mapped to `vim.lsp.buf.hover`. So, it depends on your configuration.
